### PR TITLE
	Do not show traceback and as error when issue connecting to indexer occurs in metadata creation

### PR DESCRIPTION
**Pull request by duramato**
"Fixes" stuff like this showing the full traceback and at the same time making them not submitable but still showing up to the user, as warning
```
2016-05-06 06:10:31 POSTPROCESSER :: [6cb78fe] Unable to connect to theTVDB while creating meta files - skipping - error Connection error HTTPConnectionPool(host='thetvdb.com', port=80): Max retries exceeded with url: /api/F9C450E78D99172E/series/268592/en.xml (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -2] Nome ou servi\xc3\xa7o desconhecido',)) while loading URL http://thetvdb.com/api/F9C450E78D99172E/series/268592/en.xml
Traceback (most recent call last):
  File "/home/home/SickRage/sickbeard/metadata/kodi_12plus.py", line 258, in _ep_data
    myShow = t[ep_obj.show.indexerid]
  File "/home/home/SickRage/lib/tvdb_api/tvdb_api.py", line 953, in __getitem__
    self._getShowData(key, self.config['language'], True)
  File "/home/home/SickRage/lib/tvdb_api/tvdb_api.py", line 848, in _getShowData
    self.config['url_seriesInfo'] % (sid, getShowInLanguage)
  File "/home/home/SickRage/lib/tvdb_api/tvdb_api.py", line 636, in _getetsrc
    raise tvdb_error(e)
tvdb_error: Connection error HTTPConnectionPool(host='thetvdb.com', port=80): Max retries exceeded with url: /api/F9C450E78D99172E/series/268592/en.xml (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -2] Nome ou servi\xc3\xa7o desconhecido',)) while loading URL http://thetvdb.com/api/F9C450E78D99172E/series/268592/en.xml
```

### CODE REVIEW

**Comment by fernandog** [comment]
Approved

**Comment by fernandog** [comment]
Approved

File: sickbeard/metadata/mede8er.py [hunk 1]
```diff
@@ -373,19 +383,23 @@ def write_show_file(self, show_obj):
...
             nfo_file = io.open(nfo_file_path, 'wb')
 
-            data.write(nfo_file, encoding="utf-8", xml_declaration=True)
+            data.write(nfo_file, encoding='utf-8', xml_declaration=True)
             nfo_file.close()
             helpers.chmodAsParent(nfo_file_path)
         except IOError as e:
-            logger.log(u"Unable to write file to " + nfo_file_path + " - are you sure the folder is writable? " + ex(e),
+            logger.log(u'Unable to write file to {path} - '
+                       u'are you sure the folder is writable? {exception}'.format
```
Comment (p0psicles) [inline sickbeard/metadata/mede8er.py]: exception!r ?

Comment (labrys) [inline sickbeard/metadata/mede8er.py]: You know, until now I had never looked at the purpose of `ex`?  With unicode formatted logging there's absolutely no need for it.

**Comment by p0psicles** [comment]
Approved

**Pull request merged**